test/Test.hs

```diff
@@ -259,6 +259,13 @@ main = hspec $ do
       defs <- getDefinitions doc pos
       liftIO $ defs `shouldBe` [Location (doc ^. uri) (mkRange 28 0 28 7)]
 
+  describe "getTypeDefinitions" $
+    it "works" $ runSession "hie" fullCaps "test/data/renamePass" $ do
+      doc <- openDoc "Desktop/simple.hs" "haskell"
+      let pos = Position 20 23  -- Quit value
+      defs <- getTypeDefinitions doc pos
+      liftIO $ defs `shouldBe` [Location (doc ^. uri) (mkRange 10 15 10 19)]  -- First constructor
```

Comment:
Should the type definition request not return the location of the type, not the constructor?

Reply:
The intention was that I need the function `getTypeDefinition` to implement tests for HIE.
Adding tests currently impossible, since it is kind of a hen and egg problem, I require tests to implement the feature in HIE, and I require a working HIE in this repo for the tests to succeed. Maybe we should add the tests later?

Reply:
What I had in mind for the test expectation was that the location be this
```suggestion
      liftIO $ defs `shouldBe` [Location (doc ^. uri) (mkRange 10 5 10 12)]  -- data Command = ...
```

But I think this PR should be fine without this test for now. I think we can just comment this test out and put it back in whenever the feature is merged into HIE.